### PR TITLE
For #16132 - Rename normal mode menu item, remove telemetry

### PR DESCRIPTION
**Pull request by ekager**
oops missed these 

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/tabtray/MultiselectSelectionMenu.kt [hunk 1]
```diff
@@ -28,15 +26,13 @@ class MultiselectSelectionMenu(
                 context.getString(R.string.tab_tray_multiselect_menu_item_bookmark),
                 textColorResource = R.color.primary_text_normal_theme
             ) {
-                context.components.analytics.metrics.track(Event.TabsTraySaveToCollectionPressed)
```
Comment (gabrielluong) [inline app/src/main/java/org/mozilla/fenix/tabtray/MultiselectSelectionMenu.kt:31]: Should we also be removing the telemetry from `Event.kt`?

**Review by gabrielluong — APPROVED** [review]

File: app/src/main/java/org/mozilla/fenix/tabtray/MultiselectSelectionMenu.kt (hunk 1 above)
Comment (ekager) [reply app/src/main/java/org/mozilla/fenix/tabtray/MultiselectSelectionMenu.kt:31]: Good point - I ended up reusing the event and moving it to the new button that does the actual collection saving (had to rework some of the metrics and add some kdocs too)

**Comment by ekager** [comment]
Need to update UI tests tom

File: app/src/main/java/org/mozilla/fenix/tabtray/MultiselectSelectionMenu.kt [hunk 2]
```diff
@@ -28,15 +26,13 @@ class MultiselectSelectionMenu(
...
                 onItemTapped.invoke(Item.BookmarkTabs)
             },
 
             SimpleBrowserMenuItem(
                 context.getString(R.string.tab_tray_multiselect_menu_item_close),
                 textColorResource = R.color.primary_text_normal_theme
             ) {
-                context.components.analytics.metrics.track(Event.TabsTrayShareAllTabsPressed)
```
Comment (gabrielluong) [inline app/src/main/java/org/mozilla/fenix/tabtray/MultiselectSelectionMenu.kt:39]: Also curious why we were removing the telemetry tracking for these browser menu?

Comment (ekager) [reply app/src/main/java/org/mozilla/fenix/tabtray/MultiselectSelectionMenu.kt:39]: This telemetry was just wrong -> this menu item actually closes selected tabs, but we don't have a telemetry ticket for these events yet

**Pull request merged**